app/build.gradle

```diff
@@ -1,13 +1,16 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
```

Comment:
- `'kotlin-android'` should be below `'com.android.application'`
- we use default annotation processing so remove `apply plugin: 'kotlin-kapt'`
- We are not using any of Kotlin extensions, so remove `apply plugin: 'kotlin-android-extensions'`

Reply:
> * `'kotlin-android'` should be below `'com.android.application'`

Alright, I'll reorder the code for this.

> * we use default annotation processing so remove `apply plugin: 'kotlin-kapt'`

`kapt` is required by the project since the code converted to Kotlin requires this to process the annotations for libraries like Room, Glide etc. So this is to be kept.

> * We are not using any of Kotlin extensions, so remove `apply plugin: 'kotlin-android-extensions'`

Yeah, this one was automatically added with Kotlin configuration. Not required now, so will remove this.